src/zcl/definition/cluster.ts

```diff
@@ -1912,6 +1912,12 @@ const Cluster: {
             viessmannCustom13: {ID: 0x404c, type: DataType.enum8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             viessmannCustom14: {ID: 0x404d, type: DataType.bitmap8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             viessmannCustom15: {ID: 0x404e, type: DataType.bitmap8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
+            wiserSmartPriorityLevel: {ID: 0xe000, type: DataType.uint8},
```

Comment:
Please add the `manufacturerCode` to these attributes (like the line above).

Reply:
I just tested this, unfortunately these devices don't set the MS bit in their attributes/reports, so this ends up breaking the converter. I could instead decode these as numbers directly in the converter if that is the preferred way for non-standard behaviour.

Reply:
This is preferred indeed, manufacturer attributes are only allowed anymore when carrying a manufacturerCode

Reply:
Thanks for the guidance, I have moved the attribute decode now into the converters, so these attribute conversions aren't required any longer.